### PR TITLE
Revert "fix: correct show bgp summary output typo references (#13395)"

### PR DESCRIPTION
**Pull request by cyw233**
This reverts commit 4fd776d1a1598c5c5072b4604de66575ecd32984.

### Description of PR

The https://github.com/sonic-net/sonic-utilities/pull/3375 PR has been reverted (revert PR: https://github.com/sonic-net/sonic-utilities/pull/3423) because it will impact many automation scripts that the community users may have developed for years, so we need to revert this change as well.

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

### CODE REVIEW

File: tests/route/test_route_flow_counter.py [hunk 1]
```diff
@@ -214,9 +214,4 @@ def _get_nexthop(self, duthost, ipv6):
         else:
             cmd = 'show ip bgp summary'
         parse_result = duthost.show_and_parse(cmd)
```
Comment (wenyiz2021) [inline tests/route/test_route_flow_counter.py:216]: just want to catch up on this.
after merging the revert PR, what is the change for output of `show ip bgp sum` that we need to to remove below lines?

Comment (wenyiz2021) [reply tests/route/test_route_flow_counter.py:216]: nvm, I understand now, it's fixing a typo in `neighbor`

**Review by wenyiz2021 — APPROVED** [review]
LGTM

**Review by yejianquan — APPROVED** [review]
LGTM

**Pull request merged**